torch/distributions/dirichlet.py

```diff
@@ -75,6 +76,13 @@ def log_prob(self, value):
     def mean(self):
         return self.concentration / self.concentration.sum(-1, True)
 
+    @property
+    def mode(self):
+        concentrationm1 = self.concentration - 1
+        mode = (concentrationm1 / concentrationm1.sum(-1, True)).clamp(0)
+        mode = mode * torch.where((self.concentration <= 1).any(-1, True), nan, 1.)
```

Comment:
ditto: should we handle boundary values? If we don't then someone will file a bug report that `Dirichlet(torch.tensor([0.1, 100])).mode` is not `torch.tensor([1.0, 0.0])` as expected 🤷

Reply:
Fixed in 7b4fec3e74e9efff255e019f398fa8696c8c0438.